lib/graphlient/query.rb

```diff
@@ -44,11 +45,22 @@ def indent
     end
 
     def get_args_str(args)
-      args.detect { |arg| arg.is_a? Hash }.map do |k, v|
+      hash_args(args).map do |k, v|
         "#{k}: #{get_arg_value_str(v)}"
       end.join(', ')
     end
 
+    def hash_args(args)
+      args.detect { |arg| arg.is_a? Hash }
+    end
+
+    def get_non_param_arg(args)
+      non_param_arg = args.detect { |arg| arg.is_a? Symbol }.to_s
+      return if non_param_arg.empty?
+      @skip_curly_wrapper = true
```

Comment:
This is side effect, I don't think it's right because calling this twice would produce different results. Refactor the class to remove the local completely or pass it through methods.